pushnotifications/src/main/java/com/pusher/pushnotifications/featureflags/FeatureFlagManager.kt

```diff
@@ -10,7 +10,7 @@ enum class FeatureFlag {
  */
 object FeatureFlagManager {
   private val flags: Map<FeatureFlag, Boolean> = mapOf(
-    FeatureFlag.DELIVERY_TRACKING to false
+    FeatureFlag.DELIVERY_TRACKING to true
```

Comment:
Would it make more sense if we remove the feature flag from the code now?
Or is there a chance that we will disable `DELIVERY_TRACKING` in the future?

Reply:
Removing is an option.

I'd like to see this `FeatureFlagManager` to eventually be synchronized with the server so we can remotely enable/disable features. So, with that in mind, for now, setting it to `true` is good enough.